ui/lib/apps/DebugAPI/apilist/ApiList.module.less

```diff
@@ -1,5 +1,19 @@
-.collapse_panel:not(:last-child) {
-  border-bottom: 1px solid #eee;
+.collapse_panel {
```

Comment:
It looks pretty dangerous for us to rely on such styles. Our styles may be broken in future again when antd is upgraded.

Reply:
Yes, we have a lot of hacking of antd style in our project.

Reply:
Snapshot tests seems to be really helpful in these cases!